test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -540,7 +540,17 @@ public class HubEndpointTests
                 client.Dispose();
 
                 // Ensure the client channel is empty
-                Assert.Null(client.TryRead());
+                var message = client.TryRead();
```

Comment:
I had to "fix" this test since it kept failing with the close message thing.

/cc @JamesNK